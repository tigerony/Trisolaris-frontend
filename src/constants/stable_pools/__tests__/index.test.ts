import _ from 'lodash';
import * as STABLE_POOLS from '../index'

describe('constants/stable_pools/index.ts', () => {
    test('Stable Pools', () => {
        _.map(STABLE_POOLS, (tokens, stablePoolName) => {
            expect({[stablePoolName]: tokens}).toMatchSnapshot()
        })
    })
});