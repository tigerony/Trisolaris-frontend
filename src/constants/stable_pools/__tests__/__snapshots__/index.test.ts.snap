// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants/stable_pools/index.ts Stable Pools 1`] = `
Object {
  "TriPoolOneThreeThreeSeven": Object {
    "1313161554": Array [
      Token {
        "address": "0xe3520349F477A5F6EB06107066048508498A291b",
        "chainId": 1313161554,
        "decimals": 18,
        "name": "Dai Stablecoin",
        "symbol": "DAI",
      },
      Token {
        "address": "0xB12BFcA5A55806AaF64E99521918A4bf0fC40802",
        "chainId": 1313161554,
        "decimals": 6,
        "name": "USD Coin",
        "symbol": "USDC",
      },
      Token {
        "address": "0x4988a896b1227218e4A686fdE5EabdcAbd91571f",
        "chainId": 1313161554,
        "decimals": 6,
        "name": "Tether USD",
        "symbol": "USDT",
      },
    ],
    "137": Array [
      Token {
        "address": "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063",
        "chainId": 137,
        "decimals": 18,
        "name": "Dai Stablecoin",
        "symbol": "DAI",
      },
    ],
    "43113": Array [
      Token {
        "address": "0x0000000000000000000000000000000000000000",
        "chainId": 43113,
        "decimals": 18,
        "name": "Dai Stablecoin",
        "symbol": "DAI",
      },
    ],
    "43114": Array [
      Token {
        "address": "0xbA7dEebBFC5fA1100Fb055a87773e1E99Cd3507a",
        "chainId": 43114,
        "decimals": 18,
        "name": "Dai Stablecoin",
        "symbol": "DAI",
      },
    ],
  },
}
`;
